flixel/FlxSprite.hx

```diff
@@ -1207,6 +1225,7 @@ class FlxSprite extends FlxObject
 	
 	private function set_color(Color:FlxColor):Int
 	{
+		Color = 0x00FFFFFF & Color;
```

Comment:
Okay, it definitely isn't necessary. The bug I was worried about was later traced to something else now that I'm looking at it again. Reverted.